php/delivery/class-lazy-load.php

```diff
@@ -57,7 +57,7 @@ public function setup_hooks() {
 	 */
 	public function init_lazy_script_maybe() {
 
-		$flag = filter_input( INPUT_GET, 'cloudinary_lazy_load_loader', FILTER_SANITIZE_STRING );
+		$flag = ! empty( $_GET['cloudinary_lazy_load_loader'] ) ? htmlspecialchars( $_GET['cloudinary_lazy_load_loader'] ) : null;
```

Comment:
@pereirinha : should we not be using the `Utils::get_sanitized_text()` for this purpose?
https://github.com/cloudinary/cloudinary_wordpress/blob/d9263d464db608976a44681f09ec2948943e274b/php/class-utils.php#L474-L484

Reply:
@pereirinha - Maybe we can alter that and add an ability to change the callback?